services/compute.go

```diff
@@ -382,13 +383,13 @@ func (c *client) WaitForGroupDeleted(securityGroupID string) error {
 }
 
 // BindFloatingIP binds floating IP to instance
-func (c *client) BindFloatingIP(floatingIP string, instanceID string) error {
+func (c *client) BindFloatingIP(floatingIP, instanceID string) error {
 	opts := floatingips.AssociateOpts{FloatingIP: floatingIP}
 	return floatingips.AssociateInstance(c.ComputeV2, instanceID, opts).Err
 }
 
 // UnbindFloatingIP unbinds floating IP to instance
-func (c *client) UnbindFloatingIP(floatingIP string, instanceID string) error {
+func (c *client) UnbindFloatingIP(floatingIP, instanceID string) error {
```

Comment:
why you remove string here?

Reply:
it's redundant, multiple arguments with same type don't require type after each argument